src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -390,5 +405,60 @@ private async Task ProcessSearchResultsAsync(IEnumerable<SearchResultEntry> sear
                 await UpdateJobRecordAsync(cancellationToken);
             }
         }
+
+        private async Task<SearchResult> GetGroupPatients(string groupId, List<Tuple<string, string>> queryParametersList, CancellationToken cancellationToken)
+        {
+            if (!queryParametersList.Exists((Tuple<string, string> parameter) => parameter.Item1 == KnownQueryParameterNames.Id))
+            {
+                var patientIds = await GetGroupPatientIds(groupId, cancellationToken);
+                queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.Id, string.Join(',', patientIds)));
+            }
+
+            using (IScoped<ISearchService> searchService = _searchServiceFactory())
+            {
+                return await searchService.Value.SearchAsync(
```

Comment:
What happens here if we have more than 100 patient ids as part of the `queryParametersList`?